docs/clients/promtail/troubleshooting.md

```diff
@@ -166,3 +166,9 @@ behavior of Promtail is to assign a timestamp to logs at the time it read the
 entry from the tailed file. This would result in duplicate log lines being sent
 to Loki; to avoid this issue, if your tailed file has a timestamp embedded in
 the log lines, a `timestamp` stage should be added to your pipeline.
+
+## Viewing the Service Discovery web UI (in Kubernetes)
+
+When trying to troubleshoot Kubernetes service discovery, it's really useful to look at the **Service Discovery** and **Targets** web UI. There's
+more info on that here:
+https://github.com/grafana/loki/blob/master/docs/getting-started/troubleshooting.md#troubleshooting-targets
```

Comment:
```suggestion
When trying to troubleshoot Kubernetes service discovery, it's really useful to look at the **Service Discovery** and **Targets** web UI. 

More info can be found in the [Getting Started Troubleshooting](../../../getting-started/troubleshooting.md#troubleshooting-targets) page
https://github.com/grafana/loki/blob/master/docs/getting-started/troubleshooting.md#troubleshooting-targets
```

Reply:
Thanks for submitting this @kerneljack!

Not totally sure I go that link path right but I think it's better if we use a relative link vs a full URL

Reply:
Yeah, I agree, that's better than the full link for sure

Reply:
Hi @kerneljack could you accept the suggested change or make a similar change and push?

Also to merge this PR we would need you to agree to the CLA

Thanks!

Reply:
@kerneljack will you be able to do the needful to get this PR merged?
@slim-bean to not keep this PR stranded for too long can you please open another PR if we do not hear back by end of this week?

Reply:
@sandeepsukhani thanks for the nudge, unfortunately I am unable to sign the CLA right now so it looks like you may have to close the PR for now. Sorry about that.

Reply:
@sandeepsukhani you take over that PR. Thanks @kerneljack.